src/app/credExplorer/WeightConfig.js

```diff
@@ -90,23 +91,41 @@ export class WeightConfig extends React.Component<Props, State> {
   }
 
   render() {
+    const {expanded} = this.state;
     return (
-      <div
-        style={{
-          display: "flex",
-          flexWrap: "wrap",
-          justifyContent: "space-between",
-        }}
-      >
-        <EdgeConfig
-          edgeWeights={this.state.edgeWeights}
-          onChange={(ew) => this.setState({edgeWeights: ew}, () => this.fire())}
-        />
-        <NodeConfig
-          nodeWeights={this.state.nodeWeights}
-          onChange={(nw) => this.setState({nodeWeights: nw}, () => this.fire())}
-        />
-      </div>
+      <React.Fragment>
+        <button
+          onClick={() => {
+            this.setState(({expanded}) => ({
```

Comment:
No preference from me, but you can (I believe) collapse these three lines if you like.